docs/kubernetes/domains-google-cloud-infrastructure.md

```diff
@@ -0,0 +1,53 @@
+# Domains in Google Cloud infrastructure.
+Every change in domain needs to be done manually. To pass traffic through domain name you need to modify `ingress.yml` and `webserver-php-fpm.yml` which serves as webserver and listens on domain names.
+
+## Adding Domain
+Let's say we want to listen on new domain name.
+
+Open [ingress.yml](/project-base/kubernetes/ingress.yml) file and add new domain block into `spec -> rules`:
+
+```
+// kubernetes/ingress.yml
+
+rules:
+    -   host: ~
+        http:
+            paths:
+            -   path: /
+                backend:
+                    serviceName: webserver-php-fpm
+                    servicePort: 8080
+```
+*Note*: ${DOMAIN_INDEX} is a index of domain in ingress.yml
+
+Create secrets from TSL certificate in `ingress-patch.yml` in `kubernetes/kustomize/overlays/production/ingress-patch.yml`:
```

Comment:
The file needs to be created? Now, it does not exist

Reply:
It does.